Assets/Tests/InputSystem/Plugins/PlayerInputTests.cs

```diff
@@ -149,11 +150,11 @@ public void PlayerInput_CanUseSameActionsForUIInputModule()
         eventSystemGO.SetActive(true);
         playerGO.SetActive(true);
 
-        Assert.That(actions.FindActionMap("Gameplay").enabled, Is.True);
-        Assert.That(actions.FindActionMap("UI").enabled, Is.True);
-        Assert.That(actions["UI/Navigate"].controls, Is.Empty);
-        Assert.That(actions["UI/Point"].controls, Is.EquivalentTo(new[] { mouse.position }));
-        Assert.That(actions["UI/Click"].controls, Is.EquivalentTo(new[] { mouse.leftButton }));
+        Assert.That(player.actions.FindActionMap("Gameplay").enabled, Is.True);
```

Comment:
Maybe insert a new line after this one and add a comment why there is a split now?